src/libraries/System.Net.Http/tests/StressTests/HttpStress/NuGet.config

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
-    <!-- Add public nuget feed. -->
-    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
+    <clear />
```

Comment:
Why do you clear this? We need what's in repository wide nuget.config

Reply:
If this is true, then you should not have this file at all. We don't allow nuget.org. Let me delete this and we'll see the fallout. FYI, this already broke other partners within the runtime repo.

Reply:
Let's try. BTW, `/azp run runtime-libraries stress-http` is the magic to run the pipeline.

Reply:
One more thing, can we import packages from nuget.org to `dotnet-public` if they are missing? What's the process for that?